common/changes/@itwin/core-backend/nick-time-model-extents-queries_2022-09-22-18-25.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "time getmodelextents per modelid",
```

```suggestion
      "comment": "",
```
since #4330 merged already with this changefile